File: resolucoes/JavaScript/lista01/ex003.js
const prompt = require('prompt-sync')();

function main() {
    const nome = prompt("Qual é o seu nome?")
    const salario = prompt("Qual seu salario?")
    prompt(`O funcionário ${nome} tem um salário de ${salario} em Junho.`)
}

module.exports = {
    main
}

File: resolucoes/Lua/ex003.lua
-- Lê o nome do funcionário
print("Qual e seu nome?")
nome = io.read()

-- Lê o salário do funcionário
print("Qual seu salario?")
salario = io.read()

-- Mostra a mensagem na tela
print("O funcionario " .. nome .. " tem um salario de " .. salario .. " em julho")